examples/analog_hamiltonian_simulation/01_Introduction_to_Aquila.ipynb

```diff
@@ -7,175 +7,205 @@
     "tags": []
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    time_max = np.pi/(2*amplitude_value)  # s
</pre><p>There are three atoms now, so the enhancement factor is sqrt(3). This means we need to evolve only for pi / sqrt(3) Rabi phase. I think correctly, this line should</p><p> </p><pre class="ql-syntax" spellcheck="false">time_max = np.pi/(np.sqrt(3)*amplitude_value)  # s
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/191/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>I was using $Detuning = Omega$ and in that case the enhancement factor is not sqrt(3) but sqrt(4). And for the same reason, the final states are ggg, ggr, grg, rgg. </p><p> </p><p>But let me keep it simple by setting Detuning = 0, then the enhancement factor is sqrt(3), and the final state is ggr, grg, rgg. </p><p> </p><p>Let me know what you think. </p>